src/ui/components/Tour/Tour.tsx

```diff
@@ -200,6 +200,12 @@ const Tour: React.FC = () => {
                       showBreakpointAdd &&
                       showBreakpointEdit &&
                       oneClickLogs}
+
+                    {!showConsoleNavigate &&
+                      showBreakpointAdd &&
+                      !showBreakpointEdit &&
+                      oneClickLogs}
```

Comment:
This looks broken.
```js
                    {!showConsoleNavigate &&
                      showBreakpointAdd &&
                      showBreakpointEdit &&
                      oneClickLogs}

                    {!showConsoleNavigate &&
                      showBreakpointAdd &&
                      !showBreakpointEdit &&
                      oneClickLogs}
```

Reply:
Good call, new PR -> https://github.com/replayio/devtools/pull/9118#pullrequestreview-1407152012